jobs/ScheduleBackupJob.js

```diff
@@ -49,7 +50,7 @@ class ScheduleBackupJob extends BaseJob {
               .startBackup(_.pick(jobData, 'instance_id', 'type', 'trigger'))
               .catch(errors.Conflict, errors.UnprocessableEntity, (err) => {
                 logger.error('Some other operation already in progress on this instance:', err);
-                return retry(() => this.reScheduleBackup(job.attrs.data), {
+                return retry(() => this.reScheduleBackup(job.attrs.data, job.attrs.repeatInterval), {
```

Comment:
So, reScheduleBackup is done when backup fails with Conflict and UnprocessableEntity. When does UnprocessableEntity error comes and what about actual backup failure?